hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1128,3 +1129,44 @@ def hpat_pandas_series_le_impl(self, other):
         return hpat_pandas_series_le_impl
 
     raise TypingError('{} The object must be a pandas.series and argument must be a number. Given: {} and other: {}'.format(_func_name, self, other))
+
+
+@overload_method(SeriesType, 'unique')
+def hpat_pandas_series_unique(self, sorted=False):
```

Comment:
@akharche We had many internal discussions about `sorted` parameter. Finally, my opinion was rejected. Please ignore this parameter, return values as it was before, not in the way described in Pandas documentation.
**Important**: Please add `Note` to main documentation (not developer only. before line 1139) about this changed behavior. Some thin like `Note: Return values order sorted by values`